pkg/tsdb/influxdb/flux/executor_test.go

```diff
@@ -152,6 +154,43 @@ func TestExecuteGrouping(t *testing.T) {
 	})
 }
 
+func TestAggregateGrouping(t *testing.T) {
+	ctx := context.Background()
+
+	t.Run("Grouping Test", func(t *testing.T) {
+		runner := &MockRunner{
+			testDataPath: "aggregate.csv",
+		}
+
+		dr := ExecuteQuery(ctx, QueryModel{MaxDataPoints: 100}, runner, 50)
+
+		if dr.Error != nil {
+			t.Fatal(dr.Error)
+		}
+
+		str, _ := dr.Frames[0].StringTable(-1, -1)
```

Comment:
@kylebrandt I know this breaks your brain -- but look at how pretty it is ;)

Reply:
Not when the test changes/fails. Then the diff will be a mess - so this is bad for maintainable code.